### ISSUE TITLE
Cable placed by block placer corrupts world save

### ISSUE THREAD

**Issue by BentoGambin** [refinedmods/refinedstorage#2681]
#### Issue description:
I was setting export buses to export sapplings to a Industrial foregoing block placer. I accidentally set cables to be exported, when the block placer activated it crashed the game, upon reloading the save, the cable was there but after breaking it the game crashed again, and I could load the save anymore. I tried to use nbt editor to delete the problematic cable, but could not find any cable TE at the specified (by the crash report) position. Deleting the block placer also did nothing. 
#### What happens:
Save gets corrupted.
#### What you expected to happen:
Cable gets exported to placer, gets placed, then broken and doesn't crash.
#### Steps to reproduce:
1.Place cable item on IF's Block placer
2.Make cable get placed
3.Crash
...

#### Version (make sure you are on the latest version before reporting):

- Minecraft: 1.16.1
- Forge: 1.16.1-forge-32.0.108
- Refined Storage: refinedstorage-1.9.2b
- Industrial Foregoing: industrial-foregoing-1.16.1-3.0.5-7e692e4

Does this issue occur on a server? [yes/no]
unknown
#### If a (crash)log is relevant for this issue, link it here:
This is the first crash report (upon cable getting placed): [crash-2020-09-18_00.48.31-server.txt](https://github.com/refinedmods/refinedstorage/files/5252603/crash-2020-09-18_00.48.31-server.txt)
This is the second one (upon breaking the cable after a world reload:
[crash-2020-09-18_00.55.02-server.txt](https://github.com/refinedmods/refinedstorage/files/5252610/crash-2020-09-18_00.55.02-server.txt)
And this is after realoding the game again:
[crash-2020-09-18_01.09.22-server.txt](https://github.com/refinedmods/refinedstorage/files/5252612/crash-2020-09-18_01.09.22-server.txt)

**Comment by Darkere** [comment]
You do need to prevent the block placer from placing another cable to avoid the crash.

**Comment by BentoGambin** [comment]
I already deleted the placer with nbt editor and it did nothing :( Too bad I cant get MCedit working on 1.16

**Comment by Darkere** [comment]
All three crash reports show the blockplacer placing down the block. 

What do you mean by: 
> (upon breaking the cable after a world reload

you just need to disable the placer at 267,74,-414

**Comment by BentoGambin** [comment]
I meant that the first time it crashed I was able to reload the game, and
ther was a block there, I then broke the placer and then the cable, then it
crashed. When I reload it again the placer was gone, and the cable was
sadly still there, I thought it could be a ghost block or something, then
went on breaking it with a block breaker, but no luck
I think I'm just going to try deleting the chunk.
BTW thankyou so much for the awesome mod, and also taking you time to help
me out ou a fing sunday!
Best of luck brother.

Em dom., 20 de set. de 2020 às 18:23, Darkere <notifications@github.com>
escreveu:
